src/blockedlogscommon.cpp

```diff
@@ -1446,10 +1447,21 @@ void BlockedLogsCommon::BlockFaciesLog(const std::vector<int>          & b_ind,
     for (unsigned int m = 0 ; m < n_blocks_ ; m++)
       blocked_log[m] = IMISSING;
 
-    int   max_allowed_value = 100;  // Largest allowed value (facies number).
+    int   max_allowed_value = 10000;  // Largest allowed value (facies number).
```

Comment:
En quick-fix for å håndtere store facies-nummer i Crava. Ser ut til å fungere. På sikt burde vi nok håndtere facies på en bedre måte hvis det faktisk er relevant å bruke store facies-nummer.